src/legacy_wrappers/mongo_client.js

```diff
@@ -0,0 +1,71 @@
+'use strict';
+
+const { toLegacy, maybeCallback } = require('../utils');
+
+module.exports = Object.create(null);
+Object.defineProperty(module.exports, '__esModule', { value: true });
+
+module.exports.makeLegacyMongoClient = function (baseClass) {
+  class LegacyMongoClient extends baseClass {
+    static connect(url, options, callback) {
```

Comment:
general question - I only see one instance of `connect` in api.js.  How are we differentiating between the instance and static method in the `api.js` file, and how will we make sure we test both?

Reply:
Can we add it to api.js and just comment it out like the others?  And still add a manual test

Reply:
done!